canonical/ubuntu.com
# Wd 5646 implement dashboard updates removing toggles landscape trial

## Done

- Dashboard updated according to new design in [Figma File](https://www.figma.com/file/cFiLIEDB15qRx4wwVbboct/Pro-Dashboard---Removing-toggles-23.10?type=design&node-id=75-7183&mode=design)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- visit [/dasboard](https://ubuntu-com-13162.demos.haus/pro/dashboard)
- Compare design to actual page

## Issue / Card

Fixes [#WD-5646](https://warthogs.atlassian.net/browse/WD-5646)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
